e2e/wdio/platform/tests/search.e2e-spec.ts

```diff
@@ -1,14 +1,149 @@
-import { CheckboxGroupPO } from '../pages/checkbox-group.po';
-import { refreshPage } from '../../driver/wdio';
+import {
+    browserIsIE,
+    click,
+    getAttributeByNameArr,
+    getElementArrayLength,
+    getElementSize,
+    getText,
+    getTextArr,
+    isEnabled,
+    refreshPage,
+    setValue,
+    waitElementToBePresentInDOM,
+    waitForElDisplayed
+} from '../../driver/wdio';
+import { SearchPo } from '../pages/search.po';
+import searchPageContent from '../fixtures/appData/search-page-content';
 
-describe('Search field', () => {
-    const checkboxGroupPage = new CheckboxGroupPO();
+describe('Search field', function() {
+    const searchPo: SearchPo = new SearchPo();
 
     beforeAll(() => {
-        checkboxGroupPage.open();
+        searchPo.open();
     });
 
     afterEach(() => {
         refreshPage();
     });
+
+    it('should be present and enabled', () => {
+        const arrLength = getElementArrayLength(searchPo.searchFields);
+        for (let i = 0; arrLength > i; i++) {
+            waitElementToBePresentInDOM(searchPo.searchFields, i);
+            waitForElDisplayed(searchPo.searchFields, i);
+            expect(isEnabled(searchPo.searchFields, i)).toBe(true);
+        }
+    });
+
+    it('should have search icon present and enabled', () => {
+        const arrLength = getElementArrayLength(searchPo.searchIcons);
+        for (let i = 0; arrLength > i; i++) {
+            waitElementToBePresentInDOM(searchPo.searchIcons, i);
+            waitForElDisplayed(searchPo.searchIcons, i);
+            expect(isEnabled(searchPo.searchIcons, i)).toBe(true);
+        }
+    });
+
+    it('should have placeholder text', () => {
+        const arrLength = getElementArrayLength(searchPo.searchFields);
+        const actualPlaceholders = getAttributeByNameArr(searchPo.searchFields, 'placeholder');
+        for (let i = 0; arrLength > i; i++) {
+            waitElementToBePresentInDOM(searchPo.searchFields, i);
+            waitForElDisplayed(searchPo.searchFields, i);
+            expect(actualPlaceholders[i]).toBe(searchPageContent.search_placeholder);
+        }
+    });
+
+    it('should submit term by click on search icon ', () => {
+        const arrLength = getElementArrayLength(searchPo.searchFields);
+        for (let i = 0; arrLength > i; i++) {
+            // value without suggestion
+            setValue(searchPo.searchFields, 'test', i);
+            click(searchPo.searchIcons, i);
+        }
+        expect(getText(searchPo.cozySearchResult, 0)).toContain('test');
```

Comment:
make zero a default value for `getText` second param

Reply:
done.